src/main/java/org/grizz/model/User.java

```diff
@@ -14,8 +17,13 @@
 
     @Id
     private String id;
+    @Indexed(unique = true)
+    @Size(min = 8, max = 30)
     private String login;
+    @Min(8)
     private String passwordHash;
     private Set<String> roles;
+    private int maxPlanetNumber;
```

Comment:
I dont like the idea of keeping number of planets in user data. It can easly get out of sync with increasing complexity of whole app. Much better is to calculate it live. It is quite cheap. I case of need of calculating it more often - we can always use cache for that.